.gitignore

```diff
@@ -1,5 +1,6 @@
 build/
 CMakeFiles/
+.vscode/
```

Comment:
This sort of thing should probably be in one's own global git ignore configuration file, not in individual repositories.

Reply:
OK, removed.

Reply:
Although I guess that applies to `.dir-locals.el` as well. And if you take a look at vt's .gitignore there might be even more things to clean up :- )